src/SprykerMiddleware/Zed/Process/Business/Translator/Translator.php

```diff
@@ -46,8 +46,8 @@ public function __construct(
     public function translate(array $payload): array
     {
         $result = $payload;
-        foreach ($this->dictionary as $key => $translation) {
-            $result = $this->translateKey($result, $payload, $key, $translation);
+        foreach ($this->dictionary as $key => $translations) {
```

Comment:
This change requires update to corresponding part in project, I suppose?

Reply:
@ievgenvarava https://github.com/spryker-middleware/project/pull/2